README.md

```diff
@@ -1,22 +1,6 @@
-# Taurus data model documentation
+# GEMD Documentation
 
-Deployed to [https://citrineinformatics.github.io/taurus-documentation/](https://citrineinformatics.github.io/taurus-documentation/).
-Built with [mkdocs](https://www.mkdocs.org/#mkdocs)
+GEMD stands for Generalized Expression of Materials Data (or Graph Encoding of Materials Data, if you prefer). It's an open source format developed by the fine folks at Citrine Informatics. 
```

Comment:
Has anyone else used this expansion?  I would resist guerilla rebranding.

Reply:
As far as I can tell, there are exactly three places where the acronym has _ever_ been expanded, and two of them are in slack.

Reply:
but I'll remove it